File: docs/fundamentals/runtime-libraries/snippets/System.Globalization/CultureInfo/CurrentCulture/vb/aspculture13.vb
' Visual Basic .NET Document
Option Strict On

Imports System.Globalization
Imports System.Threading

Module Example
    Public Sub S1()
        ' <Snippet13>
        Thread.CurrentThread.CurrentCulture = CultureInfo.CreateSpecificCulture(Request.UserLanguages(0))
        ' </Snippet13>
    End Sub
End Module


Public Class Request
   Private Shared langs(3) As String
   Public Shared ReadOnly Property UserLanguages As String()
      Get
         Return langs   
      End Get
   End Property    
End Class

File: docs/fundamentals/runtime-libraries/snippets/System.Globalization/CultureInfo/CurrentCulture/csharp/aspculture13.cs
using System;
using System.Globalization;
using System.Threading;

public class Example13
{
   public static void Main()
   {
      // <Snippet13>
      CultureInfo.CurrentCulture = CultureInfo.CreateSpecificCulture(Request13.UserLanguages[0]);
      // </Snippet13>
   }
}

public class Request13
{
   private static string[] langs = new string[3];

   public static string[] UserLanguages { get { return langs; } }
}
